src/plot_politeness.py

```diff
@@ -0,0 +1,120 @@
+# Lint as: python3
```

Comment:
Wait, this file appears twice in this PR?

Reply:
:/ the other one is the duplicate